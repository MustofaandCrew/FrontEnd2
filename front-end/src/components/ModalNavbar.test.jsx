import React from "react";
import renderer from "react-test-renderer";


describe("Test suits for ModalNavbar", () => {
    it("should match with snapshot", () => {
        const tree = renderer
            .create(
                <div className="modal fade" id="exampleModal" tabIndex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
                    <div className="modal-dialog">
                        <div className="modal-content">
                            <div className="modal-header">
                                <h2 className="modal-title" id="exampleModalLabel">
                                    Categories
                                </h2>
                                <button type="button" className="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                            </div>
                            <div className="modal-body">
                                
                                    <li >
                                        <a className="a-modal" href="/">
                                            uhuy
                                        </a>
                                    </li>
                            </div>
                        </div>
                    </div>
                </div>
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});