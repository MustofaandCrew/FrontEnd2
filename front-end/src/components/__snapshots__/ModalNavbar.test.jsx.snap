// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for ModalNavbar should match with snapshot 1`] = `
<div
  aria-hidden="true"
  aria-labelledby="exampleModalLabel"
  className="modal fade"
  id="exampleModal"
  tabIndex="-1"
>
  <div
    className="modal-dialog"
  >
    <div
      className="modal-content"
    >
      <div
        className="modal-header"
      >
        <h2
          className="modal-title"
          id="exampleModalLabel"
        >
          Categories
        </h2>
        <button
          aria-label="Close"
          className="btn-close"
          data-bs-dismiss="modal"
          type="button"
        />
      </div>
      <div
        className="modal-body"
      >
        <li>
          <a
            className="a-modal"
            href="/"
          >
            uhuy
          </a>
        </li>
      </div>
    </div>
  </div>
</div>
`;
